documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -2270,10 +2270,15 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 **Output Message** : Must not have duplicate name in x-ms-enum extension , make sure every x-ms-enum name unique.  
 
 **Description** : This rule will check all the swagger files with the same api-version, and ensure there is no duplicate x-ms-enum name. 
+The following cases would be disallowed :
```

Comment:
The follow cases are deemed as violation:

Reply:
Good point

Reply:
Good point.